code/controllers/shuttle_controller.dm

```diff
@@ -45,7 +45,7 @@ datum/emergency_shuttle/proc/init()
 	var/list/pods_to_test = list ("1","2","3","5")//ADD NEW PODS HERE
 	for (var/i in pods_to_test)
 		var/area/pod_test = locate(text2path("/area/shuttle/escape_pod[i]/station"))
-		if (area_in_map(pod_test))
+		if (pod_test.x)
```

Comment:
Add a comment, because this doesn't make a lot of sense at the first glance

What I'd do is keep the proc in, and just change it to ``return area.x``, but that may be suboptimal